docs(tutorials): reviewed content MTA-1876 (#525)
scaleway/docs-content: tutorials/proxmox-softraid/index.mdx

  <Lightbox src="scaleway-proxmox_installer-06.webp" width="550" alt="" />

> 9. Configure the **Management Network**. This will be the network interface on which the Proxmox webinterface and APIs are available. It is recommended to configure it on the primary email address of your Dedibox.

Reviewer: ```suggestion
9. Configure the **Management Network**. This will be the network interface on which the Proxmox web interface and APIs are available. We recommend you configure it on the primary email address of your Dedibox.
```
